entc/gen/template/builder/setter.tmpl

```diff
@@ -32,7 +32,7 @@ in the LICENSE file in the root directory of this source tree.
 	}
 
 	{{/* Avoid generting nillable setters for nillable types. */}}
-	{{ if and (not $f.Type.Nillable) (or $f.Optional $f.Default) (not (and $updater $f.UpdateDefault)) }}
+	{{ if and (not (or $f.Type.Nillable $f.Nillable)) (or $f.Optional $f.Default) (not (and $updater $f.UpdateDefault)) }}
```

Comment:
Not sure about this. Can you explain? 

The idea with `and`-ing `{{ not $f.Type.Nillable }}` was to not generate `SetNillable<F>` for `nillable` values. e.g.

```
SetIntSlice(nil) // Valid, and no need for SetNillableIntSlice.
SetStructPtr(nil) // Valid, and no need for SetNillableStructPtr.
```